indexer/services/comlink/src/controllers/api/v4/social-trading-controller.ts

```diff
@@ -0,0 +1,109 @@
+import { stats } from '@dydxprotocol-indexer/base';
+import {
+  SubaccountFromDatabase,
+  SubaccountTable,
+  SubaccountUsernamesFromDatabase,
+  SubaccountUsernamesTable,
+} from '@dydxprotocol-indexer/postgres';
+import express from 'express';
+import {
+  checkSchema,
+  matchedData,
+} from 'express-validator';
+import {
+  Controller, Get, Query, Route,
+} from 'tsoa';
+
+import { getReqRateLimiter } from '../../../caches/rate-limiters';
+import config from '../../../config';
+import { NotFoundError } from '../../../lib/errors';
+import { handleControllerError, checkIfValidDydxAddress } from '../../../lib/helpers';
+import { rateLimiterMiddleware } from '../../../lib/rate-limit';
+import { handleValidationErrors } from '../../../request-helpers/error-handler';
+import ExportResponseCodeStats from '../../../request-helpers/export-response-code-stats';
+import { subaccountInfoToTraderSearchResponse } from '../../../request-helpers/request-transformer';
+import { TraderSearchRequest, TraderSearchResponse } from '../../../types';
+
+const router: express.Router = express.Router();
+const controllerName: string = 'social-trading-controller';
+
+@Route('trader')
+class SocialTradingController extends Controller {
+  @Get('/search')
+  async searchTrader(
+    @Query() searchParam: string,
+  ): Promise<TraderSearchResponse> {
+    if (checkIfValidDydxAddress(searchParam)) {
+
+      const subaccounts: SubaccountFromDatabase[] = await
+      SubaccountTable.findAll({
+        address: searchParam,
+        subaccountNumber: 0,
+      }, [], {});
```

Comment:
nit: remove the last option, the default option has readreplica: true, this is kinda hard to figure out but you can see here:

https://github.com/dydxprotocol/v4-chain/blob/03859377bd1a3395a24eba7a8e84c3fa9fb37bc2/indexer/packages/postgres/src/constants.ts#L120, and `USE_READ_REPLICA` is set to true in comlink
```suggestion
      }, []);
```